clang/test/Driver/cxx23-modules-print-library-module-manifest-path.cpp

```diff
@@ -0,0 +1,40 @@
+// Test that -print-library-module-manifest-path finds the correct file.
+
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+// RUN: touch %t/libc++.so
+// RUN: split-file %s %t
+// RUN: cd %t
+
+// RUN: %clang -print-library-module-manifest-path \
+// RUN:     -stdlib=libc++ \
+// RUN:     --sysroot=%t \
+// RUN:     --target=x86_64-linux-gnu 2>&1 \
+// RUN:   | FileCheck libcxx-no-module-json.cpp
+
+// RUN: touch %t/modules.json
+// RUN: %clang -print-library-module-manifest-path \
+// RUN:     -stdlib=libc++ \
+// RUN:     --sysroot=%t \
+// RUN:     --target=x86_64-linux-gnu 2>&1 \
+// RUN:   | FileCheck libcxx.cpp
+
+// RUN: %clang -print-library-module-manifest-path \
+// RUN:     -stdlib=libstdc++ \
+// RUN:     --sysroot=%S/Inputs/cxx23_modules \
```

Comment:
Does `%S/Inputs/cxx23_modules` exist?

Reply:
No this is a left-over when it used on-disc files. Nice catch.